.github/workflows/test-integrations.yml

```diff
@@ -295,7 +294,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        envoy-version: ["1.27.0"]
+        envoy-version: ["1.27.2"]
```

Comment:
Looks like this was shortened since last time due to a split of int tests for nightlies: https://github.com/hashicorp/consul/commit/d3837e389cde0e59bcf51da61de4273eae14330f
https://github.com/hashicorp/consul/commit/587663dbcbd16464fd7f3c4a01bbfe486cb89351

I think we'll need to update those and the Windows variant. Trying to hit everything that exists now, I'm getting:
```shell
rg 'envoy-version: \[' -t yaml --hidden
.github/workflows/nightly-test-integrations.yml
68:          # envoy-version: ["1.24.10", "1.25.9", "1.26.4", "1.27.0"]
102:        envoy-version: ["1.24.10", "1.25.9", "1.26.4", "1.27.0"]

.github/workflows/nightly-test-integrations-1.15.x.yml
71:          # envoy-version: ["1.22.11", "1.23.12", "1.24.10", "1.25.9"]
105:        envoy-version: ["1.22.11", "1.23.12", "1.24.10", "1.25.9"]

.github/workflows/nightly-test-integrations-1.16.x.yml
71:          # envoy-version: ["1.24.10", "1.25.9", "1.26.4", "1.27.0"]
105:        envoy-version: ["1.23.12", "1.24.10", "1.25.9", "1.26.4"]

.github/workflows/nightly-test-integrations-1.17.x.yml
71:          # envoy-version: ["1.24.10", "1.25.9", "1.26.4", "1.27.0"]
105:        envoy-version: ["1.23.12", "1.24.10", "1.25.9", "1.26.4"]

.github/workflows/test-integrations.yml
264:          # envoy-version: ["1.24.10", "1.25.9", "1.26.4", "1.27.0"]
298:        envoy-version: ["1.27.0"]

.github/workflows/test-integrations-windows.yml
57:        envoy-version: [ "1.27.0" ]
```

🧠 🌩️ Side note: I've been thinking recently about how we can avoid this sort of issue w/ Envoy version maintenance, and the best idea I have ATM is a single file that controls all matrices of Envoy versions across release branches in one place.

Reply:
Great catch! Thanks for the archaeology. I was wondering why we'd "dropped" the additional Envoy versions from the tests.

w.r.t. your side note, I think get exactly what you're saying. I made a tiny `bash`+`jq` tool that takes an input JSON config file and makes the applicable changes to each branch. Happy to share/sync on it if you'd like. I think the config file might be that "one place" you're talking about, in spirit, if not in practice.

Reply:
> w.r.t. your side note, I think get exactly what you're saying. I made a tiny bash+jq tool that takes an input JSON config file and makes the applicable changes to each branch. Happy to share/sync on it if you'd like. I think the config file might be that "one place" you're talking about, in spirit, if not in practice.

That sounds great, thank you! Feels like at least a nice mistake prevention mechanism if not also a minor toil reduction. I'll throw 30m on your calendar.